package.json

```diff
@@ -137,14 +137,17 @@
     }
   },
   "lint-staged": {
-    "*.js": [
-      "prettier --write",
-      "eslint --max-warnings=0 --fix",
-      "git add"
-    ],
-    "*.md": [
-      "remark --quiet --frail"
-    ]
+    "linters": {
+      "*.js": [
+        "prettier --write",
+        "eslint --max-warnings=0 --fix",
+        "git add"
+      ],
+      "*.md": [
+        "remark --quiet --frail"
+      ]
+    },
+    "ignore": ["lib/utils/parseCalcExpression/parser.js"]
```

Comment:
This file is ignored in `.prettierignore` and `.eslintignore`.

Reply:
It was not ignored if I committed from Sourcetree.
So I made this change.